editor/assets/stylesheets/main.scss

```diff
@@ -25,6 +25,14 @@ body.toplevel_page_gutenberg {
 	svg {
 		fill: currentColor;
 	}
+
+	ul {
+		list-style-type: disc;
```

Comment:
Should these be generic across the entire editor canvas or specific to the blocks we're implementing?